.github/ISSUE_TEMPLATE/content-edit-request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Content Edit Request
+about: Suggest spelling, grammar, and other writing edits
+---
+
+# What Currently Exists
+
+> Please copy what is currently written here and specify exactly where the requested edit exists.
```

Comment:
I added this to the committed issue - but the language is written in a way that instructs the person to copy what is in this section, not from the page they are referencing. 

Put a section before this to specific page